test/general/version.pass.cpp

```diff
@@ -15,14 +15,10 @@
 
 #include "support/test_config.h"
 
-#include _PSTL_TEST_HEADER(execution)
-
-#include <oneapi/dpl/pstl/onedpl_config.h>
-
-#include "support/utils.h"
+#include _PSTL_TEST_HEADER(version)
 
 #if !__has_include(<execution>)
-static_assert(_PSTL_VERSION == 11000, "");
+static_assert(_PSTL_VERSION == 14000, "");
```

Comment:
Just curious, didn't we really have a sync?

Reply:
We need to double-check which PSTL upstream version we are currently synced with.

Reply:
As far as I can tell, this is the last sync #381 (plus some minor cleanup right after).  
I was not involved in these sync efforts though so its possible I'm not great at searching for this.